huaweicloud/services/dew/resource_huaweicloud_kms_key.go

```diff
@@ -243,7 +246,7 @@ func resourceKmsKeyV1Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("key_id", v.KeyID)
```

Comment:
it's better to warp the error with multi-error

Reply:
done